config/settings_schema.json

```diff
@@ -133,6 +133,81 @@
       }
     ]
   },
+  {
+    "name": "t:settings_schema.popup_drawer.name",
+    "settings": [
+      {
+        "type": "range",
+        "id": "popup_drawer_border_width",
+        "min": 0,
+        "max": 10,
+        "step": 1,
+        "unit": "px",
+        "label": "t:settings_schema.popup_drawer.settings.popup_drawer_border_width.label",
+        "default": 0
```

Comment:
Should we default to `1` instead ? Menu and local selector and cart notification and other popup have no borders and it feels/looks odd: ![](https://screenshot.click/21-11-ca0cp-6n0p4.png)

Reply:
Good point, Ludo! Updated to 1 and we can change the default after if needed